### PR TITLE
Update docs to encourage stronger dynamic passwords

### PR DESCRIPTION
**Pull request by majormoses**
## Description

Use dynamic passwords in documentation over static ones that could be copy/pasted.

## Motivation and Context

The `password` of "password" sadly is still one of the most used passwords. If someone is going to copy/paste let it at least be a relatively secure and random password.

### CODE REVIEW

**Comment by sensu-discourse** [comment]
This pull request has been mentioned on **Sensu Community**. There might be relevant details there:

https://discourse.sensu.io/t/new-doc-create-limited-service-accounts/2510/2

**Review by nixwiz — COMMENTED** [review]
I like this change.  Thanks @majormoses !

**Review by hillaryfraley — APPROVED** [review]

**Comment by hillaryfraley** [comment]
@majormoses I added a phrase just to be explicit about the random password and propagated to all 6.x versions. Wdyt?

File: content/sensu-go/6.0/operations/control-access/create-limited-service-accounts.md [hunk 1]
```diff
@@ -41,7 +41,7 @@ A limited service account requires:
 **NOTE**: To use a service account to manage Sensu resources in more than one namespace, create a [cluster role](../rbac/#cluster-role-example) instead of a role and a [cluster role binding](../rbac/#cluster-role-binding-example) instead of a role binding.
 {{% /notice %}}
 
-1. Create a user with the username `ec2-service`:
+1. Create a user with the username `ec2-service` and a dynamically created random password:
 
    {{< code shell >}}
 sensuctl user create ec2-service --password='password'
```
Comment (hillaryfraley) [inline content/sensu-go/6.0/operations/control-access/create-limited-service-accounts.md]:
```suggestion
sensuctl user create ec2-service --password=$(head -c1M /dev/urandom | sha512sum | cut -d' ' -f1 | head -c 32)
```

File: content/sensu-go/6.1/operations/control-access/create-limited-service-accounts.md [hunk 2]
```diff
@@ -41,7 +41,7 @@ A limited service account requires:
 **NOTE**: To use a service account to manage Sensu resources in more than one namespace, create a [cluster role](../rbac/#cluster-role-example) instead of a role and a [cluster role binding](../rbac/#cluster-role-binding-example) instead of a role binding.
 {{% /notice %}}
 
-1. Create a user with the username `ec2-service`:
+1. Create a user with the username `ec2-service` and a dynamically created random password:
 
    {{< code shell >}}
 sensuctl user create ec2-service --password='password'
```
Comment (hillaryfraley) [inline content/sensu-go/6.1/operations/control-access/create-limited-service-accounts.md]:
```suggestion
sensuctl user create ec2-service --password=$(head -c1M /dev/urandom | sha512sum | cut -d' ' -f1 | head -c 32)
```

File: content/sensu-go/6.3/operations/control-access/create-limited-service-accounts.md [hunk 3]
```diff
@@ -41,7 +41,7 @@ A limited service account requires:
 **NOTE**: To use a service account to manage Sensu resources in more than one namespace, create a [cluster role](../rbac/#cluster-role-example) instead of a role and a [cluster role binding](../rbac/#cluster-role-binding-example) instead of a role binding.
 {{% /notice %}}
 
-1. Create a user with the username `ec2-service`:
+1. Create a user with the username `ec2-service` and a dynamically created random password:
 
    {{< code shell >}}
 sensuctl user create ec2-service --password='password'
```
Comment (hillaryfraley) [inline content/sensu-go/6.3/operations/control-access/create-limited-service-accounts.md]:
```suggestion
sensuctl user create ec2-service --password=$(head -c1M /dev/urandom | sha512sum | cut -d' ' -f1 | head -c 32)
```

**Pull request merged**